source/GameMode.cpp

```diff
@@ -124,6 +124,11 @@ void GameMode::dispose() {
  * @param timestep  The amount of time (in seconds) since the last frame
  */
 void GameMode::update(float timestep) {
+	// Check if need to go back to menu
+	isBackToMainMenu = sgRoot.getIsBackToMainMenu();
+	// Set needle percentage in pause menu
+	sgRoot.setNeedleAngle((float)(net->getNumPlayers() - 1) / (float)globals::MAX_PLAYERS);
+
```

Comment:
TODO the needle angle needs to be multiplied by `0.9` to line up correctly. This happens inside main menu already; that constant should probably be factored out into `Globals.h`